torch/_torch_docs.py

```diff
@@ -1500,6 +1500,58 @@ def merge_dicts(*dicts):
     False
 """)
 
+add_docstr(torch.cov,
```

Comment:
Also update _tensor_docs.py for the method variant, and torch.rst and tensors.rst, which will tell the docs where this entry should appear.